doc/nrf/external_comp/dult.rst

```diff
@@ -244,6 +249,41 @@ Asynchronous support is also necessary to report sound state changes that are tr
 
 .. rst-class:: numbered-step
 
+.. _ug_dult_motion_detector:
+
+Interacting with the motion detector
+************************************
+
+DULT motion detector is an optional feature of the DULT subsystem.
+For more details, see the `DULT Motion detector`_ section of the DULT documentation.
+You can enable the :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR` Kconfig option to support the DULT motion detector feature in your project.
+
+To integrate the motion detector feature, set the motion detector unwanted tracking accessory capability bit (:c:enum:`DULT_ACCESSORY_CAPABILITY_MOTION_DETECTOR_UT_BIT_POS`) in the accessory capabilities bitmask.
+You must do this when registering the DULT user to indicate support for this feature.
+
+To register the motion detector callbacks, use the :c:func:`dult_motion_detector_cb_register` function.
+All motion detector callbacks defined in the :c:struct:`dult_motion_detector_cb` structure are mandatory to register:
+
+* The motion detector start request is indicated by the :c:member:`dult_motion_detector_cb.start` callback.
+  After this callback is called, the motion detector events are polled periodically with the :c:member:`dult_motion_detector_cb.period_expired` callback.
+  Typical action after the motion detector start request is to power up the accelerometer and start collecting motion data.
+* The motion detector period expired event is indicated by the :c:member:`dult_motion_detector_cb.period_expired` callback.
+  This callback is called at the end of each motion detector period.
+  The :c:member:`dult_motion_detector_cb.start` callback indicates the beginning of the first motion detector period.
+  The next period is started as soon as the previous period expires.
+  The user should notify the DULT module if motion was detected in the previous period.
+  The return value of this callback is used to pass this information.
+  The motion must be considered as detected if it fulfills the requirements defined in the `DULT Motion detector`_ section of the DULT documentation.
+* The motion detector stop request is indicated by the :c:member:`dult_motion_detector_cb.stop` callback.
+  It concludes the motion detector activity that was started by the :c:member:`dult_motion_detector_cb.start` callback.
+  Typical action after the motion detector stop request is to power down the accelerometer.
+
+The motion detector is started by the DULT subsystem when the accessory is in the separated state for a sufficient amount of time.
+When the motion is detected during the motion detector active period, the DULT subsystem calls the :c:member:`dult_sound_cb.sound_start` callback to request the sound action.
+It is done to inform the non-owner that is moving the accessory, that the device might have been used to track the non-owner.
```

Comment:
@annwoj what do you think?

Reply:
If I understood the entire context correctly maybe this could be a good rewording:

`Emitted sounds help to alert the non-owner that they are carrying an accessory that does not belong to them and might be used by the original owner to track their location.`

?

Reply:
yes, that's it